admin/page/ajax-edit-berita.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+    include("../system/connection.php");
+
+    $id = $_POST["id"];
```

Comment:
Codacy found an issue: [Detected usage of a possibly undefined superglobal array index: $_POST["id"]. Use isset() or empty() to check the index exists before using it](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)